tapestry.css

```diff
@@ -52,6 +56,63 @@ body {
     margin-top: 100px;
 }
 
+#header slider-wrapper {
+    position: absolute;
+    padding-top: 100px;
+    padding-left: 100px;
+}
+
+#slider-wrapper {
+    padding-left: 10%;
+}
+
+.slider {
```

Comment:
Could you please change this to the ID instead? Using this simple class name might cause conflicts.

Reply:
fixed in tapestry.css – removed whole class assignment from the createDepthSlider function.